File: 1317-convert-integer-to-the-sum-of-two-no-zero-integers/1317-convert-integer-to-the-sum-of-two-no-zero-integers.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        lst=[]
        for i in range(1,n):
            val1=str(i)
            val2=str(n-i)
            if '0' not in val1 and '0' not in val2:
                lst.append(i)
                lst.append(n-i)
                return lst
        

File: 1317-convert-integer-to-the-sum-of-two-no-zero-integers/1317-convert-integer-to-the-sum-of-two-no-zero-integers.java
import java.util.*;

class Solution {
    public int[] getNoZeroIntegers(int n) {
        for (int i = 1; i < n; i++) {
            String val1 = String.valueOf(i);
            String val2 = String.valueOf(n - i);

            if (!val1.contains("0") && !val2.contains("0")) {
                return new int[]{i, n - i};
            }
        }
        return new int[0];
    }
}
